pkg/models/load_balancer.go

```diff
@@ -52,6 +57,7 @@ type CreateNetworkLoadBalancerResp struct {
 
 type NetworkLoadBalancerResp struct {
 	ID          int          `json:"id" tf:"id,computed"`
```

Comment:
Can we not reuse ID itself here ? why LBID again ?

Reply:
removed